data/monthly.csv

```diff
@@ -1,262 +1,334 @@
-2014-10,3.779999999999999804600747665972448885440826416015625
-2014-11,4.12000000000000010658141036401502788066864013671875
-2014-12,3.479999999999999982236431605997495353221893310546875
-2015-01,2.9900000000000002131628207280300557613372802734375
-2015-02,2.87000000000000010658141036401502788066864013671875
-2015-03,2.8300000000000000710542735760100185871124267578125
-2015-04,2.609999999999999875655021241982467472553253173828125
-2015-05,2.850000000000000088817841970012523233890533447265625
-2015-06,2.779999999999999804600747665972448885440826416015625
-2015-07,2.839999999999999857891452847979962825775146484375
-2015-08,2.770000000000000017763568394002504646778106689453125
-2015-09,2.660000000000000142108547152020037174224853515625
-2015-10,2.339999999999999857891452847979962825775146484375
-2015-11,2.089999999999999857891452847979962825775146484375
-2015-12,1.9299999999999999378275106209912337362766265869140625
-2016-01,2.279999999999999804600747665972448885440826416015625
-2016-02,1.9899999999999999911182158029987476766109466552734375
-2016-03,1.729999999999999982236431605997495353221893310546875
-2016-04,1.9199999999999999289457264239899814128875732421875
-2016-05,1.9199999999999999289457264239899814128875732421875
-2016-06,2.589999999999999857891452847979962825775146484375
-2016-07,2.819999999999999840127884453977458178997039794921875
-2016-08,2.819999999999999840127884453977458178997039794921875
-2016-09,2.9900000000000002131628207280300557613372802734375
-2016-10,2.979999999999999982236431605997495353221893310546875
-2016-11,2.54999999999999982236431605997495353221893310546875
-2016-12,3.589999999999999857891452847979962825775146484375
-2017-01,3.29999999999999982236431605997495353221893310546875
-2017-02,2.850000000000000088817841970012523233890533447265625
-2017-03,2.87999999999999989341858963598497211933135986328125
-2017-04,3.100000000000000088817841970012523233890533447265625
-2017-05,3.149999999999999911182158029987476766109466552734375
-2017-06,2.979999999999999982236431605997495353221893310546875
-2017-07,2.979999999999999982236431605997495353221893310546875
-2017-08,2.899999999999999911182158029987476766109466552734375
-2017-09,2.979999999999999982236431605997495353221893310546875
-2017-10,2.87999999999999989341858963598497211933135986328125
-2017-11,3.0099999999999997868371792719699442386627197265625
-2017-12,2.819999999999999840127884453977458178997039794921875
-2018-01,3.87000000000000010658141036401502788066864013671875
-2018-02,2.6699999999999999289457264239899814128875732421875
-2018-03,2.689999999999999946709294817992486059665679931640625
-2018-04,2.79999999999999982236431605997495353221893310546875
-2018-05,2.79999999999999982236431605997495353221893310546875
-2018-06,2.970000000000000195399252334027551114559173583984375
-2018-07,2.8300000000000000710542735760100185871124267578125
-2018-08,2.95999999999999996447286321199499070644378662109375
-2018-09,3
+Date,Price

```

Comment:
@gradedSystem please make sure you are not changing the table schema, i.e., column names.

Reply:
@gradedSystem Fixed